frontend/javascripts/admin/user/experience_modal_view.js

```diff
@@ -410,7 +410,7 @@ class ExperienceModalView extends React.PureComponent<Props, State> {
         ) : null}
         <SelectExperienceDomain
           disabled={false}
-          mode="tags"
```

Comment:
The mode property is no longer needed here. It served the purpose to be able to select the entered text/experience domain although it is not included in the suggestions. As I implemented this behaviour manually, this is no longer required. Please note that the approach "changing mode to 'tags' in `task_create_form_view`" does change the behaviour of the select and thus this is not an option for the  `task creation view`.